obs2ioda-v2/src/cxx/netcdf_error.cc

```
@@ -0,0 +1,24 @@
+#include <sstream>
+#include "netcdf_error.h"
+
+namespace Obs2Ioda {
+
+    int netcdfErrorMessage(
```

cppcheck (2.10-2)

[style] The function 'netcdfErrorMessage' is never used.

(unusedFunction)